packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionList.tsx

```diff
@@ -44,6 +48,7 @@ export function VersionList(props: VersionListProps): ReactElement {
               />
             );
           })}
+          {props.isPaging && <LoadingSpinner className="vc-spinner-entry-list"></LoadingSpinner>}
```

Comment:
Always use self-closing JSX element when there are no children.

```suggestion
          {props.isPaging && <LoadingSpinner className="vc-spinner-entry-list" />}
```